pkg/plugins/plugins.go

```diff
@@ -10,7 +10,11 @@ import (
 	"github.com/rs/zerolog/log"
 )
 
-const localGoPath = "./plugins-local/"
+const (
+	localGoPath     = "./plugins-local/"
+	PluginTypeWASM  = "wasm"
```

Comment:
See my comment in https://github.com/traefik/traefik/pull/10189/files#r1371295003